app/src/store/reducers.test.js

```diff
@@ -0,0 +1,57 @@
+import { moviesReducer } from './reducers';
+import { initialState } from './initialState';
+import { actionType } from './actionType';
+
+describe('movies reducer', () => {
+    it('not matched action type', () => {
+        let value = moviesReducer(undefined, {
+            type: actionType.SET_SORTER
+        })
+
+        expect(value).toEqual(initialState.movies);
+    })
+
+    it('get movies', () => {
+        let value = moviesReducer(['movie1','movie2'], {
+            type: actionType.GET_MOVIES

```

Comment:
u can test through your actions instead

Reply:
I'm not sure that I understood you properly: you mean to test actions here? But we have separate file `actionCreators.test.js` with actions tests